app/src/main/java/com/endava/myendava/fragments/FaqFragment.java

```diff
@@ -37,15 +44,23 @@
     @Inject
     FaqsViewModel mFaqViewModel;
 
+    @Inject
+    KeyboardHelper mKeyboardHelper;
+
     @BindView(R.id.faq_recycler_view)
     RecyclerView mFaqRecycler;
 
     @BindView(R.id.progress_bar)
     ProgressBar mProgressBar;
```

Comment:
"usually there is no space between attributes of same type (applies to @Inject and @BindView also)"